nautobot_netbox_importer/tests/fixtures/3.7/summary.txt

```diff
@@ -120,9 +120,18 @@ fcb61481-0da8-5412-b68b-2463016a017d P2-12B | ['Rack R204 (Row 2) and power pane
 Total validation issues: 48
 - Content Types Mapping Deviations: ----------------------------------------------------------------
   Mapping deviations from source content type to Nautobot content type
+account.usertoken => account.usertoken | Disabled with reason: Nautobot content type: `account.usertoken` not found
```

Comment:
Is this different from Nautobot's `users.token`?

Reply:
Do we want to import user tokens from NetBox to Nautobot? I would rather skip importing passwords and tokens.